src/Chapter1.hs

```diff
@@ -541,8 +541,12 @@ value after "=" where the condition is true.
 
 Casual reminder about adding top-level type signatures for all functions :)
 -}
-
-mid x y z = error "mid: not implemented!"
+mid :: Int -> Int -> Int -> Int
+mid x y z
+    | (x < y && y < z) || (z < y && y < x) = y
+    | (y < x && x < z) || (z < x && x < y) = x
+    | x == y || y == z                     = y
+    | otherwise = z
```

Comment:
Nice implementation! I think you can make it even shorter if you use `<=` instead of `<`. It looks like this should make one guard redundant :slightly_smiling_face: